modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -78,17 +83,25 @@ const setupEvents = (editor: Editor, uiComponents: RenderUiComponents) => {
   });
 };
 
-const render = (editor: Editor, uiComponents: RenderUiComponents, rawUiConfig: RenderUiConfig, backstage: UiFactoryBackstage, args: RenderArgs): ModeRenderInfo => {
+// TINY-9226: When introducing two sinks, the dialog mothership should be attached to the ui
```

Comment:
Is this still accurate? We are attaching it below is all

Reply:
It's probably a bit confusing, but TINY-9226 is a future task. It hasn't been done yet. I was trying to mark what it would need to change. Same with the other TINY-9226.